cloudformation/template.yml

```diff
@@ -5,44 +5,55 @@ Metadata:
     ParameterGroups:
       -
         Label:
-          default: Project Configuration
+          default: Project Metadata
         Parameters:
           - Project
           - Environment
           - Slug
-      -
-        Label:
-          default: EC2 Instance Configuration
+      - Label:
+          default: Required Parameters
         Parameters:
           - VPC
           - SubnetIds
           - KeyName
-          - AMI
-          - InstanceTypes
+      -
+        Label:
+          default: AMIs and Instance Types (Advanced)
+        Parameters:
+          - GpuAMI
+          - CpuAMI
+          - GpuInstanceTypes
+          - CpuInstanceTypes
+      - Label:
+          default: Batch Compute Parameters (Advanced)
+        Parameters:
           - MinimumVCPUs
           - DesiredVCPUs
           - MaximumVCPUs
+          - SpotFleetBidPercentage
       -
         Label:
-          default: Docker Image Configuration
+          default: Docker Image Parameters (Advanced)
         Parameters:
-          - RepositoryName
+          - GpuRepositoryName
+          - CpuRepositoryName
           - ImageTag
       -
         Label:
-          default: Docker Container Configuration
+          default: Docker Container Parameters (Advanced)
         Parameters:
           - InstanceVCPUs
           - InstanceMemory
-    ParameterLabels:
       Project:
         default: Name
       Environment:
         default: Environment
       Slug:
         default: Slug
-      AMI:
-        default: AMI
+      GpuAMI:
+        default: AMI (GPU)
+      CpuAMI:
+        default: AMI (CPU)
```

Comment:
While Lewis mentioned that they typically use the GPU-enabled AMI for CPU jobs, I figured it wouldn't hurt to include it as an optional parameter.